src/Chat.tsx

```diff
@@ -113,6 +119,21 @@ export class Chat extends React.Component<ChatProps, {}> {
         });
     }
 
+    private handleKeyDownCapture(evt: React.KeyboardEvent<HTMLDivElement>) {
+        const target = evt.target as HTMLElement;
+        const chatviewPanelElement = findDOMNode(this.chatviewPanel);
+        const tabIndex = getTabIndex(target);
+
+        if (
+            target === findDOMNode(this.chatviewPanel)
```

Comment:
target === chatviewPanelElement

Reply:
Thanks for catching this, I would remove the `chatviewPanelElement` line instead, so people won't look at 2 places to find out what that `===` is doing.